code/modules/mob/living/carbon/human/species.dm

```diff
@@ -278,19 +281,34 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 			else	//Entries in the list should only ever be items or null, so if it's not an item, we can assume it's an empty hand
 				C.put_in_hands(new mutanthands())
 
+	if(ROBOTIC_LIMBS in species_traits)
+		for(var/obj/item/bodypart/B in C.bodyparts)
+			B.change_bodypart_status(BODYPART_ROBOTIC) // Makes all bodyparts robotic.
+			B.render_like_organic = TRUE
```

Comment:
Should probably have its own limb subtypes, but i can't see an easy way to do so with a trait.

Reply:
What do you mean by this?

Reply:
I don't think it's out of scope, if you want to add a species, you have to be prepared to do the grunt work

Reply:
Let's consider I actually do this, do I do it in this PR, or a separate one?

Reply:
It's a lot easier to have it as it's own but it's your poison :^)

Reply:
Doing it in this one will save you time but will also prevent the improvement from going in, in case this isn't merged. Whatever you like best, i guess.